.github/workflows/main.yml

```diff
@@ -0,0 +1,256 @@
+name: 'main'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  build-binaries:
+    runs-on: ubuntu-latest
+    env:
+      CC: gcc-9
+      CXX: g++-9
+
+    steps:
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update -qq
+        sudo apt install -y g++-9 build-essential cmake tclsh ant default-jre swig google-perftools libgoogle-perftools-dev python3 python3-dev uuid uuid-dev tcl-dev flex libfl-dev
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 1
+
+    - uses: actions/setup-python@v2
+      with:
+         python-version: '3.7'
+
+    - name: Build binaries
+      run: |
+        cd Surelog && make PREFIX=$PWD/../image/ release install -j $(nproc) && cd ..
```

Comment:
It would be nice to have this in some script or make target, to be able to easily build and test everything locally. Preferably typing one command would build whatever's needed and run a specified test (something that was possible before).

Reply:
Added build_binaries.sh script.